app/controllers/chat_controller.rb

```diff
@@ -278,6 +278,15 @@ def set_user_last_read
       .where(user: current_user, chat_channel: @chat_channel)
       .update_all(last_read_message_id: params[:message_id])
 
+    mention_notifications = Notification
+      .where(user: current_user)
+      .where(notification_type: Notification.types[:chat_mention])
+      .where("data LIKE ?", "%\"chat_channel_id\":#{@chat_channel.id}%")
```

Comment:
Will this be in any way performant on sites with loads of `Notification` records? 🤔

Reply:
The plan is to create a `chat_mentions` table that stores the notification_id and go off that in the future. B/c yes I bet this query is slow AF

Reply:
probably not too bad though. It's scoped to the user AND `chat_mention` notification type

Reply:
Oh I should add `unread: true`

Reply:
thereeeee we go